src/jit/codegencommon.cpp

```diff
@@ -4852,11 +4844,27 @@ void CodeGen::genCheckUseBlockInit()
 
         regSet.rsSetRegsModified(RBM_EDI);
 
-        if  (maskCalleeRegArgMask & RBM_ECX)
+#ifndef UNIX_AMD64_ABI
```

Comment:
if a #ifdef has a #else, I would prefer that it be #ifdef, not #ifndef. That avoids the logical double negative for the #else case.